Command/DriverLockCommand.php

```diff
@@ -8,15 +8,15 @@
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 
 /**
  * Create a lock action
  *
  * @package LexikMaintenanceBundle
  * @author  Gilles Gauthier <g.gauthier@lexik.fr>
  */
-class DriverLockCommand extends ContainerAwareCommand
+class DriverLockCommand extends Command
```

Comment:
The `getDriver()` method calls `$this->getContainer()->get('lexik_maintenance.driver.factory')`, which will no longer work if you replace the parent class.

https://github.com/lexik/LexikMaintenanceBundle/blob/3a3e916776934a95834235e4a1d71e4595d515f5/Command/DriverLockCommand.php#L138-L141

The alternative is to extend commands from the `Command` class and use proper service injection in the command constructor. Not sure if that is compatible with Symfony 2.7–5.1 though.